src/Auth0.AuthenticationApi/Models/AssociateNewAuthenticatorResponse.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System.Collections.Generic;
+using Newtonsoft.Json;
+
+namespace Auth0.AuthenticationApi.Models
+{
+    public class AssociateNewAuthenticatorResponse
```

Comment:
```suggestion
    public class AssociateMfaAuthenticatorResponse
```

Can we also update the filename?

Reply:
Also updated filename `AssociateMfaAuthenticatorResponse`